projects/ion/src/lib/pagination/pagination.component.ts

```diff
@@ -97,6 +110,10 @@ export class IonPaginationComponent implements OnChanges, OnInit {
     this.page = page.page_number;
   }
 
+  updateIsAdvanced(): void {
+    this.isAdvanced = this.pages.length > 10;
```

Comment:
This is a magic number, you should name this number to a more significant value to help people to identify what it means. You could change it to a const and name it with something like "MIN_PAGES_TO_ADVANCED_PAG".